src/index.js

```diff
@@ -0,0 +1,40 @@
+import './styles.css';
+import homepage from './tabHome';
+import menu from './tabMenu';
+import contact from './tabContact';
+import pageLoad from './pageLoad';
+
+let activeTab = homepage();
+
+const setActive = (e) => {
```

Comment:
I would normally caution against using impure functions. In this case that's what we're using. This function is directly manipulating a global variable `activeTab`. Let's think of a way to avoid directly manipulating a global variable from inside our functions. 

More information on why pure functions [here.](https://medium.com/@jamesjefferyuk/javascript-what-are-pure-functions-4d4d5392d49c)